static/app/views/alerts/wizard/index.tsx

```diff
@@ -74,10 +75,18 @@ class AlertWizard extends Component<Props, State> {
       params: {projectId},
     } = this.props;
     const {alertOption} = this.state;
-    const metricRuleTemplate = AlertWizardRuleTemplates[alertOption];
+    let metricRuleTemplate: WizardRuleTemplate | undefined =
+      AlertWizardRuleTemplates[alertOption];
     const isMetricAlert = !!metricRuleTemplate;
     const isTransactionDataset = metricRuleTemplate?.dataset === Dataset.TRANSACTIONS;
 
+    if (
+      organization.features.includes('alert-crash-free-metrics') &&
+      metricRuleTemplate?.dataset === Dataset.SESSIONS
+    ) {
+      metricRuleTemplate = {...metricRuleTemplate, dataset: Dataset.METRICS};
```

Comment:
Nit: can be `metricRuleTemplate.dataset = Dataset.METRICS;`

Reply:
i think i want to avoid modifying `AlertWizardRuleTemplates`